### ISSUE TITLE
Pull Headings component from component library

### ISSUE THREAD

**Issue by ChrisBAshton** [bbc/simorgh#975]
**Is your feature request related to a problem? Please describe.**
Blocked on our Headings component being merged into the component library: https://github.com/BBC-News/psammead/issues/12.

Once this is merged, we should remove the internal Headings component from Simorgh and pull in the external Headings component from Psammead.

**Describe the solution you'd like**
A single PR which removes the Headings component and imports the external one.

**Testing notes**
Will Cypress tests be required or are unit tests sufficient?

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)
